### PR TITLE
Changed dropdowns in S17 Playoff Results to a button group

### PR DESCRIPTION
**Pull request by xe-nn**

### CODE REVIEW

File: pages/S17_Playoffs.md [hunk 1]
```diff
@@ -7,8 +7,8 @@ title: S17 Playoff Results
 ```sql playoffs
 select
 m.home,
-m.away,
 concat(home_wins::INT, ' - ', away_wins::INT) as series_record,
+m.away,
```
[the remark this answers is not in the document]
Comment (xe-nn) [reply pages/S17_Playoffs.md:11]: I did this way back in the summer when you guys first created the page but if you look at the page now, it has this edit already.

File: pages/S17_Playoffs.md [hunk 2]
```diff
@@ -30,8 +30,19 @@ game_mode
 from matches m
 ```
 
-<Dropdown data={leagues} name=leagues value=league />
-
-<Dropdown data={leagues} name=gamemode value=game_mode />
+<ButtonGroup name=leagues display=tabs>
```
[the remark this answers is not in the document]
Comment (xe-nn) [reply pages/S17_Playoffs.md:33]: The dropdowns are being removed. Is that what you are referring to??

File: pages/S17_Playoffs.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (OlDirtyDirty8) [reply pages/S17_Playoffs.md:11]: This PR will be the only thing that has this line moved below the concat. Unless we have a reasoning for moving it we should keep it where it was.

File: pages/S17_Playoffs.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (OlDirtyDirty8) [reply pages/S17_Playoffs.md:33]: No. Your dropdown is named leagues and the sql query above it is also named leagues. We need to have unique names for all things within this markdown file

**Review by OlDirtyDirty8 — APPROVED** [review]

**Pull request merged**